UnityProject/Assets/Scripts/Objects/FlasherBase.cs

```diff
@@ -63,24 +64,22 @@ public void FlashInRadius()
 		}
 
 		[Server]
-		public void FlashTarget(GameObject target, float time, float stunnedtime)
+		public void FlashTarget(GameObject target, float time, float stunnedtime, bool checkForProtectiveGear = true)
```

Comment:
when would you not need to check for protection?

Reply:
could be stuff that overrides protection, such as a Wizard spell, or super Flash